apidoc/Titanium/UI/View.yml

```diff
@@ -443,7 +448,12 @@ events:
       - name: force
         summary: |
             The current force value of the touch event.
-            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later.
+            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later and on Android devices.
+        type: Number
+        
+      - name: size
+        summary: |
+            The current size of the touch area. Note: This property is only available Android devices.
```

Comment:
Typo, missing "on" in "This property is only available Android devices.", also same missing note about **some** devices.